MobileVLCKit.xcodeproj/project.pbxproj

```diff
@@ -180,7 +180,17 @@
 		8D826AE2203F3CC500536EF7 /* VLCRendererItem.h in Copy Headers */ = {isa = PBXBuildFile; fileRef = 8D2CE64E203EEA47004BB7F6 /* VLCRendererItem.h */; };
 		AACBBE4A0F95108600F1A2B1 /* Foundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = AACBBE490F95108600F1A2B1 /* Foundation.framework */; };
 		CA0301D020ADFD2500452BB8 /* libMobileVLCKit.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D2AAC07E0554694100DB518D /* libMobileVLCKit.a */; };
-		CA8A2F5620B44BD600F5D3EA /* VLCTimeTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = CA8A2F5520B44BD600F5D3EA /* VLCTimeTest.swift */; };
+		CAA03D0C20C8DCDB001764DA /* libTVVLCKit.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 7D1329591BA304D900BE647E /* libTVVLCKit.a */; };
+		CAA03D1720C8DE4D001764DA /* libbz2.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = CAA03D1620C8DE4D001764DA /* libbz2.tbd */; };
+		CAA03D1920C8DE51001764DA /* libz.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = CAA03D1820C8DE51001764DA /* libz.tbd */; };
+		CAA03D1B20C8DE58001764DA /* libiconv.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = CAA03D1A20C8DE58001764DA /* libiconv.tbd */; };
```

Comment:
why do we need to add all of these frameworks?

Reply:
I had the hardest time trying to figure this out but @fkuehne mentioned [here](https://code.videolan.org/videolan/VLCKit/issues/69) that

> For the C++ errors - do you link against libc++ and all the other required libraries? See a dynamic framework target to get an idea.

The other required libraries he was talking about are these libraries. I have no idea why `MobileVLCKitTests` doesn't require these libraries but `TVVLCKitTests` does.